.dprint.json

```
@@ -0,0 +1,44 @@
+{
+  "incremental": true,
+  "typescript": {},
+  "json": {},
+  "markdown": {
+    "lineWidth": 100
+  },
+  "toml": {},
+  "includes": [
+    "docs/**/*.{ts,js,cjs,mjs,json}",
+    "**/*.{md,toml}"
+  ],
+  "excludes": [
+    "**/node_modules",
+    "**/*-lock.json",
+    ".github/*",
+    "docs/ar-SA/**",
+    "docs/-SA/**",
+    "docs/ar-SA/**",
```

Looks like a few duplicate lines snuck in:
```suggestion
    "docs/ar-SA/**",
```

---

d'oh